### DISCUSSION TITLE
wow

### DISCUSSION THREAD

**Discussion by lis0** [maximmax42/Discord-CustomRP#34] [General]
dev replies really fast

**Comment by lis0** [comment]
btw we cant 
![image](https://user-images.githubusercontent.com/79161805/130332436-43326196-da47-4201-afe4-619b2c6675f9.png)

**Reply by maximmax42** [reply]
You can't now because I transferred issues into discussions, because discussions are better for questions than issues. I dunno why I only now enabled them.

**Reply by lis0** [reply]
ok

**Comment by maximmax42** [comment]
Well ye I get a notification about stuff right as someone posts them, so if I can answer right away, I do.

**Reply by lis0** [reply]
phone?

**Reply by maximmax42** [reply]
Nah I'm on PC most of the time.

**Reply by lis0** [reply]
ok
